charts/gatekeeper/values.yaml

```diff
@@ -145,6 +145,7 @@ controllerManager:
   priorityClassName: system-cluster-critical
   disableCertRotation: false
   tlsMinVersion: 1.3
+  certName: ""
```

Comment:
No need to modify this value, it will get updated automatically on the next release.